File: aJava/src/Leer_CSV.java
import java.io.File;
import java.io.FileNotFoundException;
import java.util.Scanner;
public class Leer_CSV {
    public static void main(String[] args) {
        System.out.println("Lectura de archivo CSV");
        
        File archivo = new File("datos.csv");

        try {
            try (Scanner lector = new Scanner(archivo)) {
                while (lector.hasNextLine()) {
                    String linea = lector.nextLine();
                    String[] campos = linea.split(";");
                    
                    System.out.println("Leyendo fila: " + campos[0] + " " + campos[1] + " " + campos[2]);
                }
            }
            System.out.println("Archivo leido exitosamente.");

        } catch (FileNotFoundException e) {
            System.out.println("Error al leer el archivo.");
        }
    }    
}

File: Python/Leer_CSV.py
import csv

inventario = []

try:
    with open("datos.csv", mode="r") as f:
        lector = csv.reader(f, delimiter=";", quotechar='"')

        for fila in lector:
            print(f"Leyendo fila {len(inventario)+1}: {fila[0]} {fila[1]} {fila[2]}")
            inventario.append(fila)
    print(f"Archivo leido exitosamente. \n{len(inventario)} filas leidas.")
except:
    print("Error al leer el archivo.")